tests/contrib/research_agents/printer.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any
```

Comment:
If everything in `research_agents` exists to just support the test, I'd say lets move it (and the test) under a `tests` subfolder to keep things tidy.

Reply:
Neeeeeever mind